experimental/scripts/configure_vscode.sh

```diff
@@ -0,0 +1,13 @@
+
+## Runs as user 
+
+install2.r languageserver lintr styler profvis httpgd
```

Comment:
`lintr` and `styler` are basic dependencies of `languageserver`, so I don't think they need to be listed here.

Reply:
:+1: